src/main/java/com/sciamus/contractanalyzer/infrastructure/adapter/mongo/MongoReportPersistenceAdapter.java

```diff
@@ -79,7 +81,13 @@ public ReportInfrastructureDTO findById(String id) {
 
         MongoQueryBuilder mongoQueryBuilder = new MongoQueryBuilder();
 
-        Pageable pageable = PageRequest.of(pageNumber, 10);
+        Pageable pageable;
+
+        if (reportFilterParameters.sortingOrder != null && reportFilterParameters.sortingProperty != null && !reportFilterParameters.sortingOrder.isBlank() && !reportFilterParameters.sortingProperty.isBlank()) {
```

Comment:
## warn

instead of using both `!=null` and `isBlank()`, we can use tool method: `Strings.isNullOrEmpty()`